content/docs/command-reference/gc.md

```diff
@@ -81,10 +81,10 @@ The default remote is cleaned (see `dvc config core.remote`) unless the
   checkpoints).
 
 - `-p <paths>`, `--projects <paths>` - if a single remote or a single cache is
-  shared among different projects (e.g. a configuration like the one described
-  [here](/doc/use-cases/shared-development-server)), this option can be used to
-  specify a list of them (each project is a path) to keep data that is currently
-  referenced from them.
+  shared among different projects (e.g. as described
+  [here](/doc/use-cases/sharing-resources-efficiently)), this option can be used
```

Comment:
it's not part of this PR, right? should the use case one go first?

Reply:
Ah yes thanks. Fixing...

Reply:
Fixed.

> should the use case one go first?

Hmm the order isn't that important I think. Since this one is ready, why not go for it. This way we don't need to worry about where to put the shared cache info. later when the shared dev server use case changes completely.